msgraph/core/middleware/request_context.py

```diff
@@ -4,16 +4,19 @@
 # ------------------------------------
 import uuid
 
+import httpx
+
 from .._enums import FeatureUsageFlag
 
 
-class RequestContext:
-    """A request context contains data that is persisted throughout the request and
-    includes a ClientRequestId property, MiddlewareControl property to control behavior
-    of middleware as well as a FeatureUsage  property to keep track of middleware used
+class GraphRequestContext:
```

Comment:
still needed?

Reply:
As per the spec, yes. Used to set `FeatureUsage` flags